src/modules/fancyzones/lib/fancyzones.rc

```diff
@@ -7,6 +7,7 @@ STRINGTABLE
 BEGIN
     IDS_SETTING_DESCRIPTION                                    "Create window layouts to help make multi-tasking easy"
     IDS_SETTING_DESCRIPTION_SHIFTDRAG                          "On: Hold Shift key or any non-primary mouse button to enable zones while dragging"
+    IDS_SETTING_DESCRIPTION_MOUSESWITCH                        "Use a non-primary mouse button to toggle zones activation while dragging"
```

Comment:
Confusing grammar, perhaps replace with: "Use a non-primary mouse button to activate zones while dragging"